site/content/blog/2023-10-12-maplocal-add-on/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Map Local Add-on"
+summary: "Allows mapping of responses to content of chosen local file."
+images:
+#- https://www.zaproxy.org/blog/2023-10-12-maplocal-add-on/images/image.png
+type: post
+tags:
+- blog
+- maplocal
+date: "2023-10-12"
+authors:
+    - Keindel (Andrey Maksimov)
+---
+
+This extension allows you to specify mappings of local resources to replace responses when using ZAP.
```

Comment:
Go into a bit more detail for why someone might want to use it?
Esp someone who has not used the same feature in Charles or Proxyman before.